tensorflow/python/ops/image_ops_impl.py

```diff
@@ -2068,7 +2068,8 @@ def _is_png(contents, name=None):
     'io.decode_image',
     'image.decode_image',
     v1=['io.decode_image', 'image.decode_image'])
-def decode_image(contents, channels=None, dtype=dtypes.uint8, name=None):
+def decode_image(contents, channels=None, dtype=dtypes.uint8, 
+                 expand_animations=True, name=None):
```

Comment:
expand_animations should come after name as it otherwise can break existing users who call all arguments positionally

Reply:
Good point. Changing the argument order now.